.gitignore

```diff
@@ -3,3 +3,4 @@
 /tests/
 /tmp-test/
 .idea
+.bash_history
```

Comment:
sorry, not that one, all the others

Reply:
Ah nice :) now I see where the skeletons are. Will revert this, and add a file .bash_history + gitignore entry to the skeletons.

Reply:
hmm we'll have another problem if the file has to exist, as tho the skeleton will be there on `ws create`, the file wont on any other person's clones

Reply:
I think you're right though, just we'd need to create the file some other way

Reply:
actually it wouldn't help on the `ws create` either. It'd be something like the enable.sh `touch .bash_history`

Reply:
otherwise if the file doesn't already exist, docker-compose would create a directory and bind that to the container, which isn't what we'd want

Reply:
yes, that's what docker-compose does... i can add `touch .bash_history` to 
src/_base/harness/config/events.yml
```
after('harness.install'): |
  #!bash
  ws enable
```
but then file does not survive a refresh/install? So I could add the file to each skeleton? I hope this makes the bash_history survive installs?

Reply:
I'd suggest https://github.com/inviqa/harness-base-php/blob/995a60cf25b378708c812ff8b926829c39d825c8/src/_base/harness/scripts/enable.sh.twig rather than the after event itself.

The file will survive in this way, as `touch` only creates the file if it doesn't exist